client/public/static/js/verification.js

```diff
@@ -7,55 +8,48 @@ $(document).ready(function () {
     console.log("Verification Code from SessionStorage:", utils.getFromStorage("verificationCode"));
   }
 
-  // Handles verification form submission logic
-  $("#verifyAccountForm").on("submit", function (event) {
-    event.preventDefault();
-    utils.clearMessageFromAuthModal(authClassesToRemove);
+ // Handles verification form submission logic
+ $("#verifyAccountForm").on("submit", function (event) {
+  event.preventDefault();
+  const verificationCode = $("#verificationCodeInput").val();
+  const username = utils.getFromStorage("username");
 
-    const form = $(this);
-    const formArray = form.serializeArray();
+  if (!username || !verificationCode) {
+    $("#verifyAccountForm").prepend(
+      `<div class="alert alert-danger">Username or verification code missing.</div>`
+    );
+    return;
+  }
 
-    const [verificationCode] = formArray.map(({ value }) => value);
-    const username = utils.getFromStorage("username");
+  fetch("/verify-account", {
+    method: "PUT",
```

Comment:
Please use PUT_ACTION instead of "PUT" if you don't mind to keep consistent across the program.